build/tasks/process.js

```diff
@@ -30,6 +30,12 @@ module.exports = function (grunt) {
       proc.kill('SIGINT');
     } else if (!proc) {
       procs[this.target] = spawn('node', [filePath]);
+
+      if (grunt.verbose) {
+        var logStream = fs.createWriteStream('services.log', {flags: 'a'});
+        procs[this.target].stdout.pipe(logStream);
+        procs[this.target].stderr.pipe(logStream);
+      }
```

Comment:
You’re right to question it.  I think a custom argument makes sense, is that can be setup through grunt?


> On Dec 16, 2015, at 2:46 PM, Jarid Margolin <notifications@github.com> wrote:
> 
> In build/tasks/process.js <https://github.com/firstopinion/rc-socket.js/pull/8#discussion_r47843037>:
> 
> > @@ -30,6 +30,12 @@ module.exports = function (grunt) {
> >        proc.kill('SIGINT');
> >      } else if (!proc) {
> >        procs[this.target] = spawn('node', [filePath]);
> > +
> > +      if (grunt.verbose) {
> > +        var logStream = fs.createWriteStream('services.log', {flags: 'a'});
> > +        procs[this.target].stdout.pipe(logStream);
> > +        procs[this.target].stderr.pipe(logStream);
> > +      }
> Hmmm. Interesting. I am not very opinionated here but want to pose a few questions.
> 
> Is it odd that it is tied to grunt.verbose but it doesn't actually write out to the console?
> Is there any argument to be made to instead utilize a custom command line argument?
> 
> ... maybe not.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/firstopinion/rc-socket.js/pull/8/files#r47843037>.
>